packages/@aws-cdk/cloudformation-include/test/test-templates/yaml/long-form-sample.yaml

```diff
@@ -0,0 +1,79 @@
+Resources:
+  RecordServiceS3Bucket:
+    Type: AWS::S3::Bucket
+    DeletionPolicy: Retain
+    Properties:
+      ReplicationConfiguration:
+        Role:
+          Fn::GetAtt:
+          - WorkItemBucketBackupRole
+          - Arn
+        Rules:
+        - Destination:
+            Bucket:
+              Fn::Join:
+              - ''
+              - - 'arn:aws:s3:::'
+                - Fn::Join:
+                  - "-"
+                  - - Ref: AWS::Region
+                    - Ref: AWS::StackName
+                    - replicationbucket
+            StorageClass: STANDARD
+          Id: Backup
+          Prefix: ''
+          Status: Enabled
+      VersioningConfiguration:
+        Status: Enabled
+  WorkItemBucketBackupRole:
+    Type: AWS::IAM::Role
+    Properties:
+      AssumeRolePolicyDocument:
+        Statement:
+        - Action:
+          - sts:AssumeRole
+          Effect: Allow
+          Principal:
+            Service:
+            - s3.amazonaws.com
+  BucketBackupPolicy:
+    Type: AWS::IAM::Policy
+    Properties:
+      PolicyDocument:
+        Statement:
+        - Action:
+          - s3:GetReplicationConfiguration
+          - s3:ListBucket
+          Effect: Allow
+          Resource:
+          - Fn::Join:
+            - ''
+            - - 'arn:aws:s3:::'
+              - Ref: RecordServiceS3Bucket
+        - Action:
+          - s3:GetObjectVersion
+          - s3:GetObjectVersionAcl
+          Effect: Allow
+          Resource:
+          - Fn::Join:
+            - ''
+            - - 'arn:aws:s3:::'
+              - Ref: RecordServiceS3Bucket
+              - "/*"
+        - Action:
+          - s3:ReplicateObject
+          - s3:ReplicateDelete
+          Effect: Allow
+          Resource:
+          - Fn::Join:
+            - ''
+            - - 'arn:aws:s3:::'
+              - Fn::Join:
+                - "-"
+                - - Ref: AWS::Region
+                  - Ref: AWS::StackName
+                  - replicationbucket
+              - "/*"
+      PolicyName: BucketBackupPolicy
+      Roles:
+      - Ref: WorkItemBucketBackupRole
```

Comment:
What's the point of having both this, and `long-form-functions.yml` and `buckets-with-parameters.yml`? It seems like they're all testing the same thing. Would be great to consolidate those into a smaller tests template.

(BTW, don't use `sample` in the name. These are all tests templates, the word 'samples' is redundant here)